docs/docs/product-documentation.md

```diff
@@ -75,54 +74,58 @@ The strategic aim of the Swapchain team is to develop an HTLC-backed cryptocurre
 
 #### 2.2.2. User scenario
 
-As a user I am either in possession of BTC, BTS or both. Let us assume I am in possession of BTCs. However, I would like to exchange BTC for BTS so that I am also in possession of BTS. Since I want to avoid counterparty risks and I do not trust people I do not know, I want to have maximum guarantees that my cryptocurrency exchange meets these requirements. Swapchain offers my desired services and satisfies my needs. Hence, I create a user account where minimal personal information is needed. Once registered, I can log in and check the orderbook for potential swap offers. Moreover, I can add my own swap offer to the orderbook. If I find a fitting swap offer or if someone accepts my previously posted swap offer, I start the ACCS by confirming the order in the orderbook. Now, the ACCS is triggered. If everything goes well, I have successfully exchanged BTC for BTS. If my swap partner tries to fool the system or the time lock runs out of time, the transaction is cancelled and I receive my refunded BTC.
+As a user I am either in possession of BTC, BTS or both. Let us assume I am in possession of BTCs. However, I would like to exchange BTC for BTS so that I am also in possession of BTS. Since I want to avoid counterparty risks and I do not trust people I do not know, I want to have maximum guarantees that my cryptocurrency exchange meets these requirements. As I already have an exchange partner, Swapchain offers my desired services and satisfies my needs. I simply access the Swapchain app and put my information, such as currency and amount I want to swap, transaction fee priority to influence the speed of my block being mined and my private and public key to sign the transaction. Subsequently, the app generates a secret hash that serves as the hashlock on the created ACCS. I pass this hashed secret to my exchange partner so he/she can put the same hash lock on the counterpart transaction. Now, the ACCS is triggered. If everything goes well, I have successfully exchanged BTC for BTS. If my swap partner tries to fool the system or the time lock runs out of time, the transaction is cancelled and I receive my refunded BTC.
 
 #### 2.2.3. UX style guide
 
 A draft of the UI mockup can be found with the following link: https://framer.com/share/swapchain-8fTM7MfGO6B5Axijk3Dy.
 
+//das müssen wir unbedingt noch ändern, da der Style ja jetzt ganz anders ist, für das Mockup haben wir aber ja besprochen, dass wir das Orderbook drinlassen 
+
```

Comment:
Habe nochmal über den UX Style Guide nachgedacht: Das ergibt bei uns eher wenig Sinn, weil wir ja keine komplexen "Wechselspiele" in der Mensch-Interface Interaktion haben. Stattdessen hat der User fast keine Buttons sondern muss alle Werte manuell eintragen. Kann also raus. ;)

Reply:
Ja, dann würde ich das denke ich auch rauslassen. Am Schluss fügen wir ja eh noch Screenshots ein und ggf. den Screencast. Dann kann der Abschnitt denke raus.